pgrx-tests/README.md

```diff
@@ -0,0 +1,9 @@
+# pgrx-tests
+
+Test framework for [`pgrx`](https://crates.io/crates/pgrx/).
+
+Meant to be used as one of your `[dev-dependencies]` when using `pgrx`.
+
+Forked off of pgrx 0.11.3 by Conrad Ludgate for the purposes of adding support for
+1. Providing options used for initialising GucContext::Backend
+2. Running tests as non-superuser
```

Comment:
I will chat with the pgrx team and see what we can do here to upstream these ideas